### PR TITLE
Database

### PR DESCRIPTION
**Pull request by skoppers**
Made the basic model for the database.

### CODE REVIEW

**Comment by feclist** [comment]
Nice data models. This can easily be integrated on the client and server api. Now we will be able to expand the UI even more :) :+1:

**Comment by hiddely** [comment]
Great tests!

File: app/models/Preset.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+ * Created by Shane on 3-5-2016.
+ */
+@Entity
+public class Preset extends Model {
+
+    @Id
+    public Long id;
+
+    public String name;
+    public int camera;
```
Comment (jorisoudejans) [inline app/models/Preset.java:22]: Shouldn't this be automatically linked to a camera object?

File: conf/routes [hunk 2]
```diff
@@ -9,6 +9,7 @@ GET     /api/scripts/:id                controllers.Script.get(id: Long)
 POST    /api/scripts/create             controllers.Script.create()
 POST    /api/scripts/:id/actions/create     controllers.Script.addAction(id: Long)
 
+
```
Comment (jorisoudejans) [inline conf/routes:12]: This?

**Comment by jorisoudejans** [comment]
Looks good

File: app/models/Preset.java (hunk 1 above)
Comment (feclist) [inline app/models/Preset.java:22]: This is something we can do when we have the camera API working

Comment (hiddely) [inline app/models/Preset.java:22]: Will be done later, when we create a camera object!

Comment (skoppers) [inline app/models/Preset.java:22]: There hasn't yet been made a camera object, and as there is still some discussion on how the object will act, for now it is linked with an integer id. Also the camera object may only be active as a live object and thus not stored in the database, for this id linking is also better.

File: conf/routes (hunk 2 above)
Comment (skoppers) [inline conf/routes:12]: oops

Comment (skoppers) [inline conf/routes:12]: just changing the game

**Pull request merged**